### PR TITLE
add documentation for using form-type

### PR DESCRIPTION
**Pull request by floriansemm**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,6 +72,41 @@ You can then use the `phone_number` mapping:
 
 This creates a `varchar(35)` column with a Doctrine mapping comment.
 
+### Forms
+
+To use phonenumbers in your forms just add the `PhoneNumberType`:
+
+	$builder->add('phoneNumber', new PhoneNumberType(), array(
```
Comment (xabbuh) [inline README.md]: You should use the registered form type service here instead.

File: README.md [hunk 2]
```diff
@@ -72,6 +72,41 @@ You can then use the `phone_number` mapping:
...
+- PhoneNumberFormat::NATIONAL
+- PhoneNumberFormat::INTERNATIONAL
+- PhoneNumberFormat::E164
+- PhoneNumberFormat::RFC3966
+
+#### Full example
+
+This example is configured for phonenumber located in germany
+
+	$builder->add('phoneNumber', new PhoneNumberType(), array(
```
Comment (xabbuh) [inline README.md]: same here

File: README.md [hunk 3]
```diff
@@ -72,6 +72,41 @@ You can then use the `phone_number` mapping:
...
+    	'label' => 'Phone',
+    	'default_region' => 'DE',
+        'format' => PhoneNumberFormat::NATIONAL
```
Comment (xabbuh) [inline README.md]: I would provide a `use` statement for the `PhoneNumberFormat` class.

**Comment by floriansemm** [comment]
@xabbuh my last commit should fix the two "defects" you mentioned.

**Comment by thewilkybarkid** [comment]
Thanks, but forms are actually there already...

It's would probably be a good idea to break the README up into separate doc files.

**Pull request closed without merging**